digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectValidator.scala

```diff
@@ -849,6 +859,15 @@ class ProjectValidator {
       Seq()
     }
   }
+  def checkSingleAdminClassOnLink(project: Project, allProjectLinks: Seq[ProjectLink]): Seq[ValidationErrorDetails] = {
+    val reversedProjectLinks = allProjectLinks.groupBy(_.linkId).filter(_._2.size > 1)
```

Comment:
En ymmärrä nimeä - miksi "reversed---"?
Tää näyttäis olevan paremminkin multiLinearLocationProjectLinksLinks tai jotain?
Vai pitäisikö sisään tulevan allProjectLinksin olla aina käännetty, eli _sen_ nimi vaatis fiksin?